signature-aggregator/aggregator/aggregator.go

```diff
@@ -121,6 +121,14 @@ func (s *SignatureAggregator) CreateSignedMessage(
 		s.metrics.FailuresToGetValidatorSet.Inc()
 		return nil, fmt.Errorf("%s: %w", msg, err)
 	}
+	s.metrics.ConnectedStakeWeightPercentage.WithLabelValues(
+		signingSubnet.String(),
+	).Set(
+		float64(
+			connectedValidators.ConnectedWeight /
+				connectedValidators.TotalValidatorWeight,
```

Comment:
This is returning a ratio, not a percentage, afaict?

Reply:
yes. i thought "percentage as a decimal" was a common enough representation go keep it that way, but i'm happy to name it to `...Ratio` or something if you'd like.

Reply:
I think I'd rather multiply by 100, I think percentage is a more readable format

Reply:
addressed in https://github.com/ava-labs/awm-relayer/pull/434/commits/503992a32b8960bcef7ae6505f940f1d5f571ce2